src/styles/main.scss

```diff
@@ -118,8 +120,8 @@ $primary-disabled: lighten( map-get($theme-colors,"secondary"), 20% );
 }
 
 .prop-heading {
-  background-color: #8C8A83 !important;
-  color: white !important;
+  background-color: $prop-heading-bg !important;
+  color: black !important;
```

Comment:
Can this be $prop-heading-color and be set to white?

Reply:
done.